weblate/lang/models.py

```diff
@@ -663,6 +663,10 @@ def is_base(self, vals: tuple[str, ...]) -> bool:
         """Detect whether language is in given list, ignores variants."""
         return self.base_code in vals
 
+    def get_source_plurals(self):
+        """Return blank source fields for pluralized new string."""
+        return ["" for _ in range(self.plural.number)]
```

Comment:
```suggestion
        return [""] * self.plural.number
```

Reply:
Applied the recommended change in the code.